notebooks/official/prediction/custom_batch_prediction_feature_filter.ipynb

```diff
@@ -0,0 +1,1586 @@
+{
```

Comment:
<p>Just omit accelerators since it's not the point of the notebook. We can simplify it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
I will simplify that! I got the training code from another notebook to bootstrap. I will simplify it :)

Reply:
Done.